sdk/storage/Azure.Storage.Common/CHANGELOG.md

```diff
@@ -1,6 +1,7 @@
 # Release History
 
 ## 12.20.0-beta.1 (Unreleased)
+- Reduce List allocations when uploading batches to table storage
```

Comment:
this should go under Other Changes in the CHANGELOG under `/sdk/tables/Azure.Data.Tables`

Reply:
Apologies, new to this. Done :)

Reply:
no worries - thanks!

Reply:
One more thing - please revert the change in this file.

Reply:
I see. Done